sparse/impl/KokkosSparse_spmv_impl_merge.hpp

```diff
@@ -69,28 +69,28 @@ struct SpmvMergeHierarchical {
 
   template <bool NONZEROS_USE_SCRATCH, bool ROWENDS_USE_SCRATCH, bool Y_USE_SCRATCH, bool CONJ>
   struct SpmvMergeImplFunctor {
-    SpmvMergeImplFunctor(const y_value_type& _alpha, const AMatrix& _A, const XVector& _x, const YVector& _y,
+    SpmvMergeImplFunctor(const y_value_type& _alpha, const AMatrix& A, const XVector& _x, const YVector& _y,
```

Comment:
Would have gone with `A_` here if possible

Reply:
Well `A_` is the member name, but I will fix this whole thing to be more consistent in any case.